spynnaker_integration_tests/test_auto_pause_and_resume_tests/too_little_memory/test_synfire_1_too_low_sdram.py

```diff
@@ -19,11 +19,11 @@
 import pytest
 from spinnaker_testbase import BaseTestCase
 from spynnaker_integration_tests.scripts import SynfireRunner
-from pacman.exceptions import PacmanPartitionException
+from pacman.exceptions import PacmanException
 
 n_neurons = 200  # number of neurons in each population
 runtime = 3000
-neurons_per_core = n_neurons / 2
+neurons_per_core = 1
```

Comment:
Does this need to be this low?  (I guess it doesn't matter as it will fit on a 48-node board, but this test won't now fit on a 4-node board, I think... ?)

Reply:
The aim of that test is to cause an exception, so it doesn't need a board in fact.